thunder_print.links.menu.yml

```diff
@@ -1,12 +1,17 @@
 
 # Print article menu items definition
 entity.print_article.collection:
-  title: 'Print article list'
-  route_name: entity.print_article.collection
+  title: 'Print articles'
+  route_name: view.print_article.print_article_list
```

Comment:
We now have two different "Collection" paths: `admin/content/print-articles` (View, in the Menu) and `admin/content/thunder_print_article` (in Breadcrumbs).

This should be unified. The view should replace the "Collection".
![Screenshot print article view](http://share.undpaul.de/bilder/jh/A_new_print_article__Site-Install_2017-04-24_06-00-02--hwvdj.png)